src/auro-flight-header.js

```diff
@@ -52,15 +52,16 @@ class AuroFlightHeader extends LitElement {
   }
 
   /**
+   * Internal function to render the day change notification.
    * @private
-   * @returns {String}
+   * @returns {String} Item to display.
    */
   flightDuration() {
-    if (this.daysChanged > 0) {
-      return `+${this.daysChanged} day${this.daysChanged > 1 ? 's' : ''}`;
-    }
+    const dayDiff = new Date(this.arrivalTime).getUTCDay() - new Date(this.departureTime).getUTCDay();
```

Comment:
I haven't tested anything, but I would make sure to carefully test this logic. Date parsing logic in JS has a lot of weird edges. Set your system timezone to a post-UTC timezone and make sure the dates show what you expect. We might want to consider bringing in a library like dayjs to make this easier.

For context: https://github.com/AlaskaAirlines/auro-pane/pull/15

Reply:
The function here mirrors the BFF the search team was using to determine the previous value that was used with this element. 

I tested this the same way you tested for the bug with auro-pane. Calculations and UI all come back as expected.